src/configurations/production.config.json

```diff
@@ -1,7 +1,7 @@
 {
     "service": {
-        "downloadUrl": "https://download.microsoft.com/download/4/D/9/4D983AEF-AF2B-48AF-AB03-78C6DE276BA1/microsoft.sqltools.servicelayer-{#fileName#}",
-        "version": "1.2.1",
+        "downloadUrl": "http://download.microsoft.com/download/8/6/F/86FEE8EC-A1D4-4A43-A946-B6CA970BBCD7/microsoft.sqltools.servicelayer-{#fileName#}",

```

Comment:
These must be `https`. Please fix & will sign off once you do

Reply:
Fixed now, thanks